guides-and-tutorials/sbtc/how-to-run-sbtc-signer.md

```diff
@@ -54,64 +56,65 @@ This notification system creates a direct event stream when:
 1. Block data publishes via ZMQ
 1. Signer processes relevant sBTC transactions
 
-### Reference configuration
+### Example
+
+```bash
+bitcoind \
+  -server \
+  -datadir=$${BITCOIN_DATA} \
+  -rpcbind=0.0.0.0 \
+  -rpcuser=$${BITCOIN_RPC_USERNAME} \
+  -rpcpassword=$${BITCOIN_RPC_PASSWORD} \
+  -rpcport=$${BITCOIN_RPC_PORT} \
+  -rpcallowip=0.0.0.0/0 \
+  -rpcallowip=::/0 \
+  -txindex \
+  -zmqpubhashblock="tcp://*:$${BITCOIN_ZMQ_PORT}" \
+  -zmqpubrawblock="tcp://*:$${BITCOIN_ZMQ_PORT}" \
+  -coinstatsindex
+```
+
+## 2. Configure your Stacks node
+
+### Minimum version
 
-<details>
-
-<summary>bitcoin.conf example</summary>
-
-```conf
-regtest=1
-
-[regtest]
-printtoconsole=1
-disablewallet=0
-txindex=1
-coinstatsindex=1
-
-# Specify a non-default location to store blockchain data.
-# blocksdir=/data/bitcoin-data
-# Specify a non-default location to store blockchain and other data.
-# datadir=/data/bitcoin-data
-
-# [network]
-# bind=0.0.0.0:18444
-discover=0
-dns=0
-dnsseed=0
-listenonion=0
-
-# [rpc]
-rpcserialversion=0
-# Accept command line and JSON-RPC commands.
-server=1
-# Accept public REST requests.
-rest=1
-rpcbind=0.0.0.0:18333
-rpcallowip=0.0.0.0/0
-rpcallowip=::/0
-rpcuser=user
-rpcpassword=password
-
-# [zmq]
-# Note that this is required for the sbtc signer to work properly.
-zmqpubhashblock=tcp://*:28332
-zmqpubrawblock=tcp://*:28332
-
-# [wallet]
-addresstype=legacy
-changetype=legacy
-fallbackfee=0.00001
+Please ensure your Stacks version is up-to-date the latest release.
+
+### Event observer
+
+You will need to add a _new_ event observer that relays information from the
+sBTC smart contracts to the sBTC signer:
+
+```toml
+[[events_observer]]
+endpoint = "sbtc-signer:8801"
+events_keys = [
+    "SM3VDXK3WZZSA84XXFKAFAF15NNZX32CTSG82JFQ4.sbtc-registry::print",
+]
 ```
 
-</details>
+### Reference configuration
+
+<!---
+TODO: This needs to be updated once the PR linked here merges.
+-->
+
+See
+[here](https://github.com/stacks-network/sbtc/blob/831b890a39ec6c9ee7dec44673843d46a7050950/docker/mainnet/nodes/stacks/Config.toml.in#L1).
 
 
-## 2. Configure your signer
+## 3. Configure your sBTC Signer
 
 The signer configuration file (`signer-config.toml`) defines the signer's
 operation parameters. The configuration sections include:
 
+### Blocklist Client Settings
+
+```toml
+[blocklist_client]
+endpoint = "http://blocklist-client:3032"
+```
+
 ### Bitcoin Connection Settings
 
 Defines how the signer connects to Bitcoin Core:
```

Comment:
The connection example bellow specifies ports `18443` and `28332` which are usually testnet-specific, they would have to be changed for mainnet

Reply:
Wouldn't let me comment on that part, I'm talking about these ports - zmq should be the same I believe (from the bitcoin configuration above in this file), but bitcoin 8332 (by default, if not changed through `.env`):
![image](https://github.com/user-attachments/assets/e0d559f8-2f7d-48fd-95f9-ef8aecf94cd1)

Reply:
Good call, editing it